.gitignore

```diff
@@ -29,3 +28,9 @@ project/plugins/project/
 # Scala-IDE specific
 .scala_dependencies
 .worksheet
+
+# frontend
+*.css.map
+.sass-cache
+bower_components
```

Comment:
As you're starting a new project, it might be worth talking to @OliverJAsh or @theefer about current thoughts about Bower. Currently with Bower you need to consider committing the dependencies due to lack of version locking, but that's not ideal so there might be an another option. Current preference seems to be for `jspm` but not sure how suitable that is for CSS dependencies.